packages/manager/modules/support/src/support.routing.js

```diff
@@ -0,0 +1,19 @@
+import component from './support.component';
+
+export const state = {
+  name: 'support',
+  redirectTo: 'support.tickets',
+  url: '/support',
+  views: {
+    '@': component.name,
+  },
+};
+
+export const registerState = /* @ngInject */ ($stateProvider) => {
```

Comment:
Looks like a duplicate from another route declaration. I don't really see the point of using a function if it's duplicated all over routing files